src/options/README

```diff
@@ -0,0 +1,225 @@
+Modules
+=======
+
+  Each options module starts with the following required attributes:
+
+    id     ... (string) ID of the module (e.g., "ARITH")
+    name   ... (string) name of the module (e.g., "Arithmetic Theory")
+    header ... (string) name of the options header to generated (e.g., "options/arith_options.h")
+
+  A module defines 0 or more options and/or aliases.
+
+  In general, each attribute/value pair is required to be in one line.
+  Comments start with # and are not allowed in attribute/value lines.
+
+
+Options
+=======
+
+  Options can be defined with the [[option]] tag, the required attributes for
+  an option are:
+
+    category ... (string) common | expert | regular | undocumented
+    type     ... (string) C++ type of the option value
+
+  Optional attributes are:
+
+    name       ... (string) option name that is used to access via
+                            options::<name>()
+    smt_name   ... (string) alternative name to access option via
+                            set-option/get-option commands
+    short      ... (string) short option name consisting of one character
+                            (no - prefix required)
+    long       ... (string) long option name (required if short is specified,
+                            no -- prefix required)
+    default    ... (string) default value of type 'type'
+    handler    ... (string) handler for parsing option values before setting
+                            option
+    predicates ... (list)   functions that check whether given option value is
+                            valid
+    includes   ... (list)   header files required by
+                            handler/predicates/notifies
+    notifies   ... (list)   notifications to call when option is set
+    links      ... (list)   additional options to set after this option is set
+    read_only  ... (bool)   true: option is should not provide a ::set method,
+                            false (default): option should provide a ::set
+                            method to set the option value
+    alternate  ... (bool)   true (default): add --no-<long> alternative option
+                            false: omit --no-<long> alternative option
+    help       ... (string) documentation (required for if category is not
+                            undocumented)
+
+  Note that if an option defines a long option name with type 'bool',
+  mkoptions.py automatically generates a --no-<long> option to set the option
+  to false.
+  This behaviour can be explicitely disabled for options with attribute
+  alternate = false.
+  More information on how to use handler, predicates and notifies can be found
+  at the end of the README.
+
+
+  Example:
+
+    [[option]]
+      name       = "outputLanguage"
+      smt_name   = "output-language"
+      category   = "common"
+      short      = ""
+      long       = "output-lang=LANG"
```

Comment:
Can you maybe add a sentence to the `long` documentation above regarding the `<long-option-name>=XXX` you used here? From the info above I would have written `long = "output-lang"`. Is the `=XXX` part required? What if I omit it?

Reply:
Done